content/events/2021-online-unconference/_index.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Nordic-RSE online unconference"
+description = "Nordic RSE – Research Software Engineers in the nordics – Unconference 29–30.June #NordiRSEunconf"
++++
+
+# Nordic RSE Unconference 2021
+
+We are planning an online unconference on June 29–30. The program will consist of your contributions and we encourage you to submit a short abstract for a discussion topic, talk, demonstration or any other type of program you would like to run.
+
+## Dates: June 29-30, 13-16 CET (3 + 3 hours)
+
+## Format and goal and scope
+
+To bring the community of research software engineers together in 2021,
```

Comment:
Maybe I'm just tired, but this sentence reads a bit weird. Maybe start with "We plan an informal unconference, in order to ... and ... ."

Reply:
Is this better?